From 05181c90cd108227b4621aa8869a202ce90b75cd Mon Sep 17 00:00:00 2001
From: Andrii Osipov <aosipov@anaconda.com>
Date: Fri, 11 Apr 2025 03:08:11 -0700
Subject: [PATCH] downgrade htbuilder lowerbound

---
 pyproject.toml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/pyproject.toml b/pyproject.toml
index c0fe64e..00bd426 100644
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -15,7 +15,7 @@ keywords = ["python", "streamlit", "ui", "data"]
 dependencies = [
   "altex>=0.2.0",
   "entrypoints >= 0.4",
-  "htbuilder >= 0.6.2",
+  "htbuilder >= 0.6.1",
   "markdownlit >= 0.0.7",
   "plotly >= 5.23.0",
   "prometheus-client >= 0.20.0",
